src/logs_db/bot.py

```
@@ -16,7 +16,7 @@
 print("db_path", db_path)
 
 
-def db_commit(query, params=()):
+def db_commit(query, params=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)